src/Shared/LayoutRenderers/AspNetSessionIdLayoutRenderer.cs

```diff
@@ -23,8 +26,12 @@ public class AspNetSessionIdLayoutRenderer : AspNetLayoutRendererBase
         /// <param name="logEvent">Logging event.</param>
         protected override void DoAppend(StringBuilder builder, LogEventInfo logEvent)
         {
+            var isSessionConfigured = true;
             var context = HttpContextAccessor.HttpContext;
-            if (context?.Session == null)
+#if ASP_NET_CORE
+            isSessionConfigured = context.Features.Get<ISessionFeature>() != null;
```

Comment:
I prefer a more specific log message and return here (so we won't need the variable)